Makefile

```diff
@@ -1166,7 +1164,7 @@ release-notes-tool:
 
 .PHONY: release-notes
 release-notes: release-notes-tool
```

Comment:
Maybe add something like this
```
	@if [ -z "${PREVIOUS_VERSION_TAG}" ]; then echo "PREVIOUS_VERSION_TAG is not set"; exit 1; fi
```

Reply:
I don't think it will work, as PREVIOUS_VERSION_TAG only need to set for RC/Beta release, other than that it will be empty.

Reply:
Makes sense